packages/nc-gui/components/FeedbackForm.vue

```diff
@@ -21,12 +15,15 @@ export default {
   computed: {
     feedbackFormHidden: {
       get() {
-        return this.$store.state.settings.feedbackFormHidden;
+        return this.$store.state.settings.feedbackForm.isShown;
```

Comment:
The form is hidden even isShown is true.

```json
{
    "url": "https://docs.google.com/forms/d/e/1FAIpQLSeTlAfZjszgr53lArz3NvUEnJGOT9JtG9NAU5d0oQwunDS2Pw/viewform?embedded=true",
    "createdAt": "2022-07-11T14:11:37.000Z",
    "isShown": true,
    "validTill": "2022-07-19T00:00:01.000Z"
}
```

This line should be `return !this.$store.state.settings.feedbackForm.isShown;` as the condition is `feedbackFormHidden`.